src/cpp/desktop/DesktopWebView.cpp

```diff
@@ -159,23 +159,26 @@ QString WebView::promptForFilename(const QNetworkRequest& request,
 
 void WebView::keyPressEvent(QKeyEvent* pEvent)
 {
+   
 #ifdef Q_OS_MAC
-   if (pEvent->key() == Qt::Key_W &&
-       pEvent->modifiers() == Qt::CTRL)
+   // on macOS, intercept Cmd+W and emit the window close signal
+   if (pEvent->key() == Qt::Key_W && pEvent->modifiers() == Qt::CTRL)
    {
-      // on macOS, intercept Cmd+W and emit the window close signal
       onCloseWindowShortcut();
+      return;
    }
-   else
+#endif
+   
+   // allow Ctrl + Shift + V to act as a paste action
+   if (pEvent->key() == Qt::Key_V && pEvent->modifiers() == Qt::CTRL + Qt::SHIFT)
```

Comment:
Do we need to do anything to prevent people from trying to use this as a custom keyboard shortcut? (Maybe not worth worrying about?)

Reply:
That's a good point. Perhaps this should be implemented as a GWT-level command / binding that delegates back to the desktop frame.